### PR TITLE
Define the literature review process

### PR DESCRIPTION
**Pull request by opsb**
This is loosely based on definition in https://github.com/emberjs/rfcs. I've kept it to a minimum for now so that it can be refined for the Elm community.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # Literature Reviews
 
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review. You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request.
```
Comment (mgold) [inline README.md]: It seems worth mentioning that elm-discuss and particularly Slack can be good for this.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+## The process
+
+* Fork the https://github.com/elm-community/literature-reviews
+* Copy TEMPLATE.md to text/my-feature-name.md
+* Fill in the template
+* Submit a pull request. As a pull request the literature review will receive feedback from the larger community, and the author should be prepared to revise it in response.
+* Eventually, Evan will decide whether the feature is a candidate for inclusion in Elm.
+* If the feature is accepted the PR will be merged.
```
Comment (mgold) [inline README.md]: I think we should go ahead an merge lit reviews that have approval from the community.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+* An RFC may be rejected by Evan after public discussion has settled and comments have been made summarizing the rationale for rejection.
```
Comment (mgold) [inline README.md]: * Please don't refer to these documents are RFCs. Although they literally are requests for comments and open discussions, the acronym has the connotation of an approved standard.

* Evan will not (necessarily, likely) review these. Approval comes from the community, separate of implementation.

**Review by mgold — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -1,3 +1,18 @@
...
 The goal of this repo is to house markdown documents exploring proposed language and library additions to the Elm language.
+
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review. You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request.
+
+## The process
+
+* Fork the https://github.com/elm-community/literature-reviews
```
Comment (BrianHicks) [inline README.md]: How about `Fork [the elm-community/literature-reviews repo](https://github.com/elm-community/literature-reviews)`?

File: README.md [hunk 5]
```diff
@@ -1,3 +1,18 @@
...
+* Copy TEMPLATE.md to text/my-feature-name.md
```
Comment (BrianHicks) [inline README.md]: Why not `my-feature-name/README.md`? That way, example code and supporting files could live next to the review and GitHub would show a nice overview on the web.

File: README.md [hunk 6]
```diff
@@ -1,3 +1,18 @@
...
+* Fill in the template
+* Submit a pull request. As a pull request the literature review will receive feedback from the larger community, and the author should be prepared to revise it in response.
+* Eventually, Evan will decide whether the feature is a candidate for inclusion in Elm.
```
Comment (BrianHicks) [inline README.md]: I don't think we should be promising this.

**Review by BrianHicks — CHANGES_REQUESTED** [review]
just tweaks :man_dancing:

File: README.md (hunk 5 above)
Comment (mgold) [inline README.md]: Oooh, I like that.

File: README.md (hunk 3 above)
Comment (opsb) [reply README.md]: copy/paste offence, I was definitely avoiding that

File: README.md (hunk 2 above)
Comment (opsb) [inline README.md]: 👍

File: README.md [hunk 7]
```diff
@@ -1,3 +1,17 @@
...
+
+## Gathering feedback before submitting
+
+It's often helpful to get feedback on your concept before getting into the detail required for a literature review.
+You may open an issue on this repo to start a high-level discussion, with the goal of eventually creating a literature review pull request. In addition you can post a message to the [elm-discuss mailing list](https://groups.google.com/forum/#!forum/elm-discuss) or chat to the folks over at [Slack](https://elmlang.herokuapp.com/).
+
+## The process
+
+* Fork the [the elm-community/literature-reviews repo](https://github.com/elm-community/literature-reviews)
+* Copy `TEMPLATE.md` to `text/my-feature-name/README.md`
```
Comment (BrianHicks) [inline README.md]: still, why `text`? It's not like this repo will contain thing *other* than literature reviews, right? It's premature namespacing, and if we need it in the future we can always shove them all into a subdirectory.

**Review by BrianHicks — CHANGES_REQUESTED** [review]

File: README.md (hunk 7 above)
Comment (opsb) [reply README.md]: Fair enough

**Review by mgold — APPROVED** [review]

**Review by BrianHicks — APPROVED** [review]

**Pull request merged**